src/allmydata/web/status.py

```diff
@@ -550,7 +550,7 @@ def events(self, req, tag):
             length = r_ev["length"]
             bytes_returned = r_ev["bytes_returned"]
             decrypt_time = ""
-            if bytes:
+            if bytes_returned is not None:
```

Comment:
I didn't look far enough to figure out if this changes the semantics or not:
- does `read_events` ever contain a value that is `None`?
- do we actually just want `if bytes_returned` here (thus ignoring / not-rendering when it's "0").

Obviously, this code can't be that well-tested because it would have always run this before ;) but this might simply be a redundant check then (i.e. if `r_ev["bytes_returned"]` can't ever be `None` anyway).

Reply:
You are right it is never ```None``` and ```if bytes_returned``` is probably a better implementation.